ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_path_selection.schema.yml

```diff
@@ -115,6 +115,28 @@ keys:
                     description: Static IPv4 addresses.
                     items:
                       type: str
+            keep_alive:
+              type: dict
+              keys:
+                auto:
+                  type: bool
+                  description: |-
+                    Adaptive keepalive and feedback interval.
+                    Set this to true to enable it.
+                    If "auto" is enabled the interval values are ignored.
+                interval:
+                  type: int
+                  description: Interval in milliseconds.
+                  min: 50
+                  max: 60000
+                  convert_types:
+                    - str
+                interval_count:
```

Comment:
lets call this failure_threshold - and add a description that says that it is the 

```
description: Failure threshold in number of intervals. Required when `interval` is set.
```

For reference:
```
Mode: DpsPathGroupConfigMode
Syntax: keepalive interval <50-60000> milliseconds [ failure-threshold <2-100> intervals ]
Syntax: keepalive interval auto

Default/No Keepalive Interval: 1000 ms
Default/No Failure Threshold: 5
```